module/japanese_mix.py

```diff
@@ -0,0 +1,110 @@
+import urllib.request
+from urllib.parse import quote
+from bs4 import BeautifulSoup
+import subprocess
+import platform
+import datetime
+import json
+import wget
+import re
+import hanziconv # https://pypi.python.org/pypi/hanziconv/0.2.1
+from re import compile as _Re
+
+_unicode_chr_splitter = _Re( '(?s)((?:[\u2e80-\u9fff])|.)' ).split
+
+def LookUp(word, data):
+    
+    result = {}
+    front_word = ""
+    back_word = ""
+    furi = ""
+    furiChild = []
+    furiList = []
+    text = ""
+    textChild = []
+    textList = []
+    reading = ""
+    cnt = 0
+    download_dir = ""
+
+    # Eliminate the end of line delimiter
+    word = word.splitlines()[0]
+    wordUrl = urllib.parse.quote(word, safe='')
+
+    jisho_Url = "http://jisho.org/search/{}".format(wordUrl)
+    jisho_Content = urllib.request.urlopen(jisho_Url).read()
+    jisho_Soup = BeautifulSoup(jisho_Content, 'lxml')
+
+    hj_Url = "https://dict.hjenglish.com/jp/jc/{}".format(wordUrl)
+    hj_Content = urllib.request.urlopen(hj_Url).read()
+    hj_Soup = BeautifulSoup(hj_Content, 'lxml')
+
+
+
+    if "download_dir" in data:
+        download_dir = data['download_dir']
+        
+    if word == "":
+        return None
+        
+    wrongSpelling = jisho_Soup.find('div', id='no-matches')
+    if wrongSpelling is not None:
+        return None
+    
+    print(" ")
+    print('<<'+word+'>>')
+    print(" ")
+
+    for i in jisho_Soup.find_all('div', class_='exact_block'):
+
+        firstBlock = i.find('div', class_='concept_light clearfix')
+        partJP = firstBlock.find('div', class_='concept_light-wrapper')
+        partEN = firstBlock.find('div', class_='concept_light-meanings')
+        status = partJP.find('div', class_='concept_light-status')
+        if(status != None):
```

Comment:
Make coding style consistent.
Use '()' or without it for all `if/else` statements,